magento/pwa-studio
# PWA-119: add product quantity stepper to cart

## Description

Adds a product quantity selector to each line item in /cart.

## Related Issue





Closes #1281 

## Acceptance 




### Verification Stakeholders

@soumya-ashok @sirugh 

### Specification



### Verification Steps


1. Add item to cart.
2. Visit /cart
3. Play with the stepper and try to break it (it should be pretty resilient)

## Screenshots / Screen Captures (if appropriate)

[![Image from Gyazo](https://i.gyazo.com/db79480aa06de3fef882e374411af392.gif)](https://gyazo.com/db79480aa06de3fef882e374411af392)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
